packages/ariakit/src/toolbar/__examples__/toolbar/style.css

```diff
@@ -0,0 +1,36 @@
+@import url("../../../button/__examples__/button/style.css");
+
+.toolbar {
+  @apply
+    flex
+    gap-1
+    border
+    p-1
+    rounded
+    border-alpha-1
+    flex-row
+    w-fit
+    items-center
+    bg-canvas-4
+    dark:bg-canvas-4-dark
+  ;
+}
+
+.button {
+  @apply
```

Comment:
is there a way to extend the button?

Reply:
What do you mean by extending the button? If we have to override a lot of styles, it's okay not to import the base button example and add everything here.